regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -49,7 +49,7 @@ var CommentReviewView = Backbone.View.extend({
 
   serialize: function() {
     return {
-      docNumber: this.docNumber,
+      doc_number: this.docNumber,
```

Comment:
This is probably a good time to resolve https://github.com/eregs/notice-and-comment/issues/23. The value here (so far) is the FR document number, which might or might not be the same as the regs.gov document ID. Do you know how the FR and regs.gov IDs are related?